agent/tcs/handler/handler.go

```diff
@@ -35,8 +36,8 @@ const (
 	defaultPublishMetricsInterval = 20 * time.Second
 
 	// The maximum time to wait between heartbeats without disconnecting
```

Comment:
Can we go ahead and add the default rwtimeout here too? even if expresed as:

`defaultRWTimeout = defaultHeartBeatJitter + defaultHeartbeatTimeout`

Reply:
Yeah, I thought about and didn't think we needed a new const for this. But, I think it makes it more explicit. I'll add that.